commands/login.go

```diff
@@ -20,13 +20,15 @@ var (
 	username      string
 	password      string
 	passwordStdin bool
+	insecure      bool
 )
 
 func init() {
 	loginCmd.Flags().StringVarP(&gateway, "gateway", "g", defaultGateway, "Gateway URL starting with http(s)://")
 	loginCmd.Flags().StringVarP(&username, "username", "u", "", "Gateway username")
 	loginCmd.Flags().StringVarP(&password, "password", "p", "", "Gateway password")
 	loginCmd.Flags().BoolVar(&passwordStdin, "password-stdin", false, "Reads the gateway password from stdin")
+	loginCmd.Flags().BoolVar(&insecure, "insecure", false, "Do not validate SSL certificate")
```

Comment:
The default should be set to `true` so it doesn't break any CI pipelines users could have.

Reply:
I think is better to add this flag in a separate PR. Refactoring should not add new functionality.